### ISSUE TITLE
Share example prompt configurations

### ISSUE THREAD

**Issue by AddictArts** [michaeldfallen/git-prompt#34]
Would you be willing to share some of the prompts you have in the README.md

For example this one
https://raw.githubusercontent.com/michaeldfallen/git-radar/master/images/detailed.png

Thank you

**Comment by josephearl** [comment]
Something like?
<img width="328" alt="screen shot 2015-08-29 at 21 37 02" src="https://cloud.githubusercontent.com/assets/2513147/9564196/248a4784-4e96-11e5-8ee7-06f49280c1b1.png">

Fork dotfiles https://github.com/JosephEarl/dotfiles and install (which is just a fork of https://github.com/michaeldfallen/dotfiles with a sub-module pointing to a private repo removed and the start of the prompt message added).

Set Terminal profile to monokai and font to Hack (it's slightly different, but close -- this should be selected after installing the dot files with the bootstrap script).

Make sure to include `.bashrc` in your `.bash_profile`:

```
if [ -f $HOME/.bashrc ]; then . $HOME/.bashrc; fi
```

The prompt used is:

```
\[\033[1;32m\]→\[\033[0m\] \[\033[1;36m\]$(basename $(pwd))/\[\033[0m\]$(git-radar --bash --fetch)
```

**Comment by AddictArts** [comment]
Yes, unfortunately this seems geared for the Linux terminal and I am on a Mac using the Mac Terminal. Git-Radar is still useful and I managed to get close. The "git:(...)" part does not appear, but the rest is there. Thanks

**Comment by josephearl** [comment]
I'm using OS X & Terminal and the "git:(...)" part appears for me when I'm in a Git repo

**Comment by michaeldfallen** [comment]
Same here, I'm using OSX, apples built in Terminal (with [TotalTerminal](http://totalterminal.binaryage.com/) wrapping it) and Zsh. 

Probably what's happening is your terminal is set up to use a different colour scheme. I use [monokai](https://github.com/michaeldfallen/dotfiles/tree/master/osx) and that has the "bright black" colour set to a dark grey. So I see `git:(` as a dark grey.

A pull request is currently open to make this configurable, #39,  so you'll be able to change the colour soon.

**Comment by mengelbrecht** [comment]
If you need another example on how to integrate git-radar into a prompt take a look at my [slimline](https://github.com/mgee/slimline) zsh theme. The git-radar information is obtained asynchronously which makes the prompt very fast even in large git repositories.

**Comment by AddictArts** [comment]
Thanks to all, I have it working now.
